scripts/local/docker-compose.yml

```diff
@@ -0,0 +1,146 @@
+# This is the docker-compose script for local development.
+# For the latest version of the docker images see:
+# https://github.com/Azure/pcs-cli/blob/master/remotemonitoring/single-vm/docker-compose.dotnet.yml
+#
+# Usage:
+# - install Docker Compose, see https://docs.docker.com/compose/install
+# - open a console and `cd` into the folder that contains this file
+# - note how the file references many environment variables, either set
+#   them in the .env file or edit this file adding the values here
+# - run `docker-compose up`
+# - open the browser at http://127.0.0.1:10443
+#
+# For more information see
+# https://github.com/Azure/azure-iot-pcs-remote-monitoring-dotnet
+
+version: "3"
+
+services:
+  reverseproxy:
+    image: azureiotpcs/remote-monitoring-nginx:1.0.0-preview.3
+    ports:
+      - "80:80"
+      - "443:443"
+    depends_on:
+      - webui
+      - auth
+      - iothubmanager
+      - devicesimulation
+      - telemetry
+      - config
+    volumes:
+      - ./:/app/certs:ro
```

Comment:
could you add a comment here, and perhaps instructions earlier, explaining what this folder should contain?

Reply:
done